src/tests/mocks/camera_mocks.hpp

```diff
@@ -8,6 +8,9 @@
 #include <components/camera/client.hpp>
 #include <components/camera/server.hpp>
 
+namespace viam {
+namespace cppsdk {
```

Comment:
Can we namespace this one layer deeper? Like viam::cppsdk::cameramocks? Otherwise functions like `fake_...` can collide when compiling the tests. (I ran into this when adding additional motor tests.)

Reply:
A case can be made that the mocks or other test code doesn't actually belong in the `viam:cppsdk` namespace at all.

I'm also not usually too keen on a complex set of "inner namespaces" due to the fact that namespaces have interactions with name lookup: https://en.cppreference.com/w/cpp/language/adl

Reply:
This still has the problem of declaring functions like `fake_properties` in the namespace for all of the tests but other tests (like motor) will also have `fake_properties` (if it follows the same pattern). Thoughts?

Reply:
Ah, good catch. I added an additional layer of namespacing for the `mock`s only, and using the appropriate namespace in associated tests. This should keep generic functions like `fake_properties` separate, and hopefully minimize namespace bloat. @acmorrow given that this is confined to tests my intuition is it's probably okay as far as the "inner namespace" issue goes but I'm curious to your thoughts.